docker-compose.yml

```diff
@@ -16,10 +14,27 @@ services:
 
   redis:
     image: redis
-    ports:
-      - "6379:6379/tcp"
     healthcheck:
       test: ["CMD", "redis-cli", "ping"]
       interval: 10s
       timeout: 3s
       retries: 10
+
+  adminer:
+    image: adminer:4.7.7-standalone
+    depends_on:
+      - mysql
+    environment:
+      ADMINER_DEFAULT_SERVER: mysql
+    ports:
+      - 8080:8080
+
+  website:
```

Comment:
How does this know what "webiste" is.

Reply:
This is just a label used internally.

Reply:
You can think of this as the internal DNS name for the container.